documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Select the `Create a simple project (skip archetype selection)` check box. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+... (Optional) Select the `Add project(s) to working set` check box to add additional projects to the working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as whether to `Resolve Workspace projects`, `Profiles`, and any `Name templates` to set a desired naming convention for the project.
++
+.Set the Module Name and Parent
+image::mvnbasic_module_configuration.png[Set the Module Name and Parent]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Module` wizard, enter the module archetype information:
+.. Ensure that the `Show the last version of Archetype only` check box is selected. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword `maven-archetype-quickstart` in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Module Archetype
+image::mvnbasic_select_module_archetype.png[Select a Module Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Module` wizard, enter the archetype details:
+.. Add a unique group ID to the `Group Id` field. Note that this ID must be different from the group ID of the included Maven project.
+.. Add the desired version number in the `Version` field. As an example, the version `0.0.1-SNAPSHOT` was used as the version number for the previously created Maven project, therefore here we can use a variation such as `0.1.1-SNAPSHOT`.
++
+.Set the Module Archetype Parameters
+image::mvnbasic_archetype_parameters.png[Set the Module Archetype Parameters]
++
+.. Click `Finish` to conclude setting up the Maven module.
+. (Optional) To change the settings for the created Maven module, expand the module name in the `Project Explorer` view and double click `pom.xml` from the expanded list. An `Overview` tab appears and all settings can be changed from this tab.
++
+.Change the Module Settings from the Overview View
+image::mvnbasic_module_pom_overview.png[Change the Module Settings from the Overview View]
++
+
+**Result:** Your new Maven module is created and appears in the `Project Explorer` view.
+
+
+[[Maven_Support]]
+== 3. Add Maven Support to an Existing Non-Maven Project
+
+The previous tasks contain instructions to create a new Maven project and Maven module. However, for an existing application that was not created with Maven support, use the following instructions to add Maven support to the non-Maven project:
+
+==== Adding Maven Support to an Existing Non-Maven Project ====
+. (Optional) To add dependencies to the project, use the following instructions:
```

Comment:
This was added based on feedback from @rawagner. As I understand it, it's an optional mini-task that users may seek to do here instead of or in addition to changing existing dependencies. Happy to take it out if this is unnecessary after all.